test_utils/model_whisper_utils.py

```
@@ -71,5 +71,5 @@ def transcribe_audio_with_whisper(ref_audio_torch: torch.Tensor, sr: int) -> str
 
     except Exception as e:
         print(traceback.format_exc())
-        logging.error(f"Failed to transcribe audio: {str(e)}")
+        logger.error(f"Failed to transcribe audio: {str(e)}")
```

Ruff (0.14.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)